tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -117,6 +137,38 @@ private void Run(BsonDocument shared, BsonDocument test)
             }
         }
 
+        // this section appears to not be alpha sorted?
+        /// <summary>
```

Comment:
The methods here don't appear to be sorted alphabetically, so I've left that as is to minimize the diff rather than sorting them and making the diff confusing.

Reply:
True. In some files methods are sorted in the order they are called. This order can sometimes be the easiest to understand when first reading a file because you can read the methods more or less in the order they will be executed.